users/service.go

```diff
@@ -619,7 +631,7 @@ func (svc service) addClientPolicy(ctx context.Context, userID string, role mgcl
 	}
 	resp, err := svc.auth.AddPolicies(ctx, &policies)
 	if err != nil {
-		return errors.Wrap(svcerr.ErrAddPolicies, err)
+		return errors.Wrap(errAddPolicies, err)
```

Comment:
Can we have some thing similar to what we have in remove policy 
```suggestion
		return errors.Wrap(svcerr.ErrAddPolicies, err)
```